### ISSUE TITLE
Mozillabuild dependency

### ISSUE THREAD

**Issue by metalcanine** [metalcanine/fx-win32k-report#22]
After #20 is resolved (either wontfix or with a commit) a cost benefit type analysis needs to be done on whether mozillabuild is needed on the machine generating reports and what versions of python, node, and npm that would luck this project into.

**Comment by metalcanine** [comment]
Since we use mozilla/bedrock running locally for testing we can't run these tests in just mozilla-build's shell as bedrock requires batch scripting and an updated version of pip, python, npm, etc.
